kafka-ui-react-app/src/lib/hooks/api/latestVersion.ts

```diff
@@ -1,13 +1,14 @@
 import { useQuery } from '@tanstack/react-query';
+import { actuatorInfoPayload } from 'lib/fixtures/actuatorInfo';
 import {
   QUERY_REFETCH_OFF_OPTIONS,
   GIT_REPO_LATEST_RELEASE_LINK,
 } from 'lib/constants';
 
 const fetchLatestVersion = async () => {
-  const data = await fetch(GIT_REPO_LATEST_RELEASE_LINK).then((res) =>
-    res.json()
-  );
+  const data = await fetch(GIT_REPO_LATEST_RELEASE_LINK)
+    .then((res) => res.json())
+    .catch(() => actuatorInfoPayload);
```

Comment:
`actuatorInfoPayload` method is a fixture method used in testing , i don't think it would be correct if the error occurred we view some Fixture data @Haarolean what should be view here ?